src/shared/Core/Trace2.cs

```diff
@@ -796,3 +840,42 @@ protected override string GetEventMessage(Trace2FormatTarget formatTarget)
         return sb.ToString();
     }
 }
+
+public class ErrorMessage : Trace2Message
+{
+    [JsonProperty("msg", Order = 7)]
```

Comment:
nit: does the `Order` attribute here mean that this field will be the 7th in the resulting JSON?

Most consumers of the JSON output don't/shouldn't care about the ordering of the fields. I just read/parse it into a dictionary and access the fields as I need them, for example.  So you could omit them here -- unless it makes testing easier (some readers might want to grep for "\"msg\":.*,\"fmt\":.*" for example, so it is ok to keep it if you think this might be the case).

Reply:
I'm fine with omitting! I included them to ensure the same order as the fields coming from Git, but since there won't typically be any human readers, it seems unnecessary.